### PR TITLE
Fix wrong action log link when add/edit article

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue #22551.

### Summary of Changes
This is an alternative and maybe better fix for issue #22551 compare to PR #22554. 

### Testing Instructions
1. See #22551 for issue description, confirm the issue
2. Apply patch, try to edit article from frontend again, check action log link, confirm it's fixed.

### CODE REVIEW

File: plugins/actionlog/joomla/joomla.php [hunk 1]
```diff
@@ -30,6 +30,13 @@ class PlgActionlogJoomla extends ActionLogPlugin
 	 */
 	protected $loggableExtensions = array();
 
+	/**
+	 * Context aliases
+	 *
+	 * @var array
+	 */
```
Comment (Quy) [inline plugins/actionlog/joomla/joomla.php:38]: Please add `@since`

**Comment by infograf768** [comment]
works fine.
Waiting for the @since to set testing Ok

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on ea1000efa4908809a1b22263c40d6cd42cb9ecd3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22562">issues.joomla.org/tracker/joomla-cms/22562</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ea1000efa4908809a1b22263c40d6cd42cb9ecd3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22562">issues.joomla.org/tracker/joomla-cms/22562</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22562">issues.joomla.org/tracker/joomla-cms/22562</a>.</sub>

**Pull request merged**